apps/nuxt3-ssr/graphql.config.json

```diff
@@ -1,6 +1,6 @@
 {
     "schema": [
-        "https://emx2.dev.molgenis.org/CatalogueOntologies/catalogue3/graphql",
-        "https://emx2.dev.molgenis.org/catalogue/catalogue3/graphql"
+        "https://emx2.dev.molgenis.org/CatalogueOntologies/catalogue/graphql",
```

Comment:
You could and probably should us https://emx2.dev.molgenis.org/CatalogueOntologies/api/graphql

Reply:
yes, i think i included this for a easy dev proxy regex, but not sure it its still needed, wil remove in separate PR